src/index.js

```diff
@@ -12,17 +12,19 @@ class App extends React.PureComponent {
   render() {
     return (
       <>
-        <Navbar />
-        <Routes>
-          <Route
-            path="/"
-            element={<BookContainer />}
-          />
-          <Route
-            path="/about"
-            element={<div>This is gonna be the about page!</div>}
-          />
-        </Routes>
+        <div className="main-holder">
+          <Navbar />
+          <Routes>
+            <Route
+              path="/"
+              element={<BookContainer />}
+            />
+            <Route
+              path="/about"
+              element={<div>This is gonna be the about page!</div>}
+            />
+          </Routes>
+        </div>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use the whole page space for the app, the white spaces around the app are just for design purposes

|Your design|How it should be|
|-|-|
|![Screenshot_20221217_193628](https://user-images.githubusercontent.com/25227022/208256551-1fb28772-17ee-4d01-87ca-bcc5189c62f2.png)|![Screenshot_20221217_194018](https://user-images.githubusercontent.com/25227022/208256578-17f3732b-157d-4c14-bac4-72a3cf174f18.png)|